recipes/pymc-bart/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pymc-bart" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pymc_bart-{{ version }}.tar.gz
+  sha256: 425814173e5c33dcf8d2226e51b0cd785c470ad6c05f411283c9284b39366e51
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - pymc >=4.0.1
+    - arviz
+    - numba
+    - matplotlib-base <=3.5.2
+
+test:
+  imports:
+    - pymc_bart
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://github.com/pymc-devs/pymc_bart
```

Comment:
```suggestion
  home: http://github.com/pymc-devs/pymc-bart
```

Reply:
Thanks @dopplershift, I will fix this upstream.